--- .gitmodules	2024-09-04 09:28:24.256801114 -0400
+++ /home/jerry/Desktop/gitmodules	2024-09-04 09:25:10.859107307 -0400
@@ -1,18 +1,18 @@
 [submodule "3rdparty/xbyak"]
 	path = 3rdparty/xbyak
-	url = ../../RPCSX/xbyak.git
+	url = https://github.com/RPCSX/xbyak.git
 [submodule "3rdparty/SPIRV-Tools"]
 	path = 3rdparty/SPIRV-Tools
-	url = ../../KhronosGroup/SPIRV-Tools.git
+	url = https://github.com/KhronosGroup/SPIRV-Tools.git
 [submodule "3rdparty/SPIRV-Headers"]
 	path = 3rdparty/SPIRV-Headers
-	url = ../../KhronosGroup/SPIRV-Headers.git
+	url = https://github.com/KhronosGroup/SPIRV-Headers.git
 [submodule "3rdparty/SPIRV-Cross"]
 	path = 3rdparty/SPIRV-Cross
-	url = ../../KhronosGroup/SPIRV-Cross.git
+	url = https://github.com/KhronosGroup/SPIRV-Cross.git
 [submodule "3rdparty/glslang"]
 	path = 3rdparty/glslang
-	url = ../../KhronosGroup/glslang.git
+	url = https://github.com/KhronosGroup/glslang.git
 [submodule "3rdparty/json"]
 	path = 3rdparty/json
-	url = ../../nlohmann/json.git
+	url = https://github.com/nlohmann/json.git
